src/Kmd.Logic.Gateway.Automation.Tool/CommandHandlerBase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Kmd.Logic.Identity.Authorization;
+
+namespace Kmd.Logic.Gateway.Automation.Tool
+{
+
+    internal abstract class CommandHandlerBase : IDisposable
+    {
+#pragma warning disable SA1401 // Fields should be private
+        protected LogicTokenProviderFactory logicTokenProviderFactory;
+        protected GatewayOptions gatewayOptions;
+#pragma warning restore SA1401 // Fields should be private
+
+        protected void Initialize(CommandBase cmd)
+        {
+            this.logicTokenProviderFactory = new LogicTokenProviderFactory(
+                new LogicTokenProviderOptions
+                {
+                    AuthorizationScope = cmd.AuthorizationScope,
+                    ClientId = cmd.ClientId,
+                    ClientSecret = cmd.ClientSecret,
+                });
+
+            this.gatewayOptions = new GatewayOptions
+            {
+                GatewayServiceUri = new Uri("https://kmd-logic-services-prod.azurefd.net"),
```

Comment:
Not required to pass this value this having default value, as off now Gateway end points are not published, passing direct end point while debug mode is fine.

Reply:
What? How Gateway endpoints are not published?

Reply:
Able see the gateway endpoint is published in PROD before it was not there :) , "gateway.kmdlogic.io/gateway/v1" this is the end point for PROD

Reply:
Could you provide whole URL? It does not work for me.

Reply:
default value is already specified
![image](https://user-images.githubusercontent.com/44639788/97973762-9721e380-1dec-11eb-9c53-e04ab15e061a.png)

Reply:
![image](https://user-images.githubusercontent.com/17142439/97974174-8317bb00-1dc7-11eb-9d09-890b2e4129f5.png)

Reply:
URL is wrong "https://gateway.kmdlogic.io/gateway/v1"

Reply:
done